sdk/keyvault/azure-keyvault-certificates/migration_guide.md

```diff
@@ -0,0 +1,288 @@
+
+credential = DefaultAzureCredential()
+
+certificate_client = CertificateClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+```
+
+### Async operations
+
+The modern `azure-keyvault-certificates` library includes a complete async API supported on Python 3.5+. To use it, you must first install an async transport, such as [aiohttp](https://pypi.org/project/aiohttp/). See [azure-core documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md#transport) for more information.
+
+Async operations are available on async clients, which should be closed when they're no longer needed. Each async client is an async context manager and defines an async `close` method. For example:
+
+```python
+from azure.identity.aio import DefaultAzureCredential
+from azure.keyvault.certificates.aio import CertificateClient
+
+credential = DefaultAzureCredential()
+
+# call close when the client is no longer needed
+client = CertificateClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+...
+await client.close()
+
+# alternatively, use the client as an async context manager
+client = CertificateClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+async with client:
+  ...
+```
+
+### Create a certificate
+
+In `azure-keyvault` you could create a certificate by using `KeyVaultClient`'s `create_certificate` method, which required a vault endpoint and certificate name. This method returned a `CertificateOperation`.
+
+```python
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.keyvault import KeyVaultClient
+
+credentials = ServicePrincipalCredentials(
+    client_id="client id",
+    secret="client secret",
+    tenant="tenant id"
+)
+
+certificate_client = KeyVaultClient(credentials)
+
+operation = certificate_client.create_certificate(
+    vault_base_url="https://my-key-vault.vault.azure.net/",
+    certificate_name="cert-name"
+)
+```
+
+Now in `azure-keyvault-certificates` you can use the `begin_create_certificate` method to create a new certificate or new version of an existing certificate with the specified name. Before creating a certificate, a management policy for the certificate can be created or a default policy can be used. This method returns a long running operation poller that can be used to check the status or result of the operation.
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.certificates import CertificateClient, CertificatePolicy
+
+credential = DefaultAzureCredential()
+
+certificate_client = CertificateClient(vault_url="https://my-key-vault.vault.azure.net/", credential=credential)
+
+create_certificate_poller = certificate_client.begin_create_certificate(
+    certificate_name="cert-name",
+    policy=CertificatePolicy.get_default()
+)
+certificate = create_certificate_poller.result()
```

Comment:
Might warrant a comment that this blocks until the operation completes (because that could take days).